elasticms-cli/src/Client/MediaLibrary/MediaLibrarySync.php

```diff
@@ -253,4 +273,36 @@ private function getMetadata(string $path): array
 
         return [];
     }
+
+    private function getFolderPath(): string
+    {
+        if (!$this->options->hashFolder) {
+            return $this->options->folder;
+        }
+
+        $folderZip = $this->getFileByHash($this->options->folder);
```

Comment:
there is no stream support? To avoid saving the zip

Reply:
Sorry this is not possible, we can discuss it